ci-operator/step-registry/openstack/provision/sriov-networknodepolicy/openstack-provision-sriov-networknodepolicy-commands.sh

```diff
@@ -172,6 +186,11 @@ fi
 
 wait_for_sriov_pods
 
+# This is only needed on ocp 4.16+
+# introduced https://github.com/openshift/sriov-network-operator/pull/887
+# u/s https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/617
+create_default_sriov_operator_config
```

Comment:
should not we check for the OCP version then?

Reply:
we could, but that's not necessary since this is `oc apply` which will overwrite the config if it already exists. This would be different if we were using `oc create`, where it would fail if the object already exists (below 4.16, or when we already ran the `openstack-provision-sriov-operator` step).